android/titanium/src/java/org/appcelerator/titanium/proxy/TiViewProxy.java

```diff
@@ -737,7 +737,32 @@ public void removeAllChildren()
 			}
 		}
 	}
+	
+	/**
+	* Returns the view by the given ID.
+	* @module.api
+	*/
+	@Kroll.method
+	public TiViewProxy getViewById(String arg)
```

Comment:
Does this work on Android? `id` is a type on iOS. Will change it here, but primarily followed other method argument lists.